modules/openapi-generator-maven-plugin/src/main/java/org/openapitools/codegen/plugin/CodeGenMojo.java

```diff
@@ -245,7 +245,7 @@
      * To treat a document strictly against the spec.
      */
     @Parameter(name = "strictSpec", property = "openapi.generator.maven.plugin.strictSpec", required = false)
-    private Boolean strictSpecBehavior;
+    private Boolean strictSpec;
```

Comment:
@jharmelink are these changes in this file intentional?

Reply:
This is actually a bugfix to get the generation working again. Not related to this change.

Reply:
The issue should have been fixed in the master. Please have a look and let us know if that's not the case